client/my-sites/plugins/plugin.jsx

```diff
@@ -29,6 +29,7 @@ import PluginSections from 'my-sites/plugins/plugin-sections';
 import pluginsAccessControl from 'my-sites/plugins/access-control';
 import EmptyContent from 'components/empty-content';
 import FeatureExample from 'components/feature-example'
+import WpcomPluginsList from 'my-sites/plugins-wpcom/plugins-list';
```

Comment:
I can't comment on it because it didn't change in this branch, but plugin.jsx has an unnecessary config import.